src/main/java/com/learn/java/collections/Set.java

```diff
@@ -0,0 +1,33 @@
+package com.learn.java.collections;
+
+public interface Set<T> {
+    /**
+     * The current amount of stored elements
+     * @return
+     */
+    int size();
+
+    /**
+     * Stores a new element in the last position
```

Comment:
It isnt required to be in the last position.
It is enough with: "Stores the element if it doesnt exists yet"